Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1857 -- "Related Sections" tweaks

Resolves [EREGCSC-1857](https://jiraent.cms.gov/browse/EREGCSC-1857)

**Description**

Adjusts styles in the "Related Sections" list for each Resources item listed in Search Page resources results or Resources page results.

**This pull request changes:**

- Creates RelatedSections component to encapsulate all logic and markup needed to display Related Sections on Search page and Resources page
- Styles for each page are different -- unique styles are untouched

**Steps to manually verify this change:**

1. Visit Search page and Resources page.  Ensure that related sections links are grouped by title.
